### ISSUE TITLE
Enable specifying alt text for book-theme logos

### ISSUE THREAD

**Issue by raebot** [jupyter-book/myst-theme#804]
A WCAG 2.1 AA audit of pages generated with book theme shows that logos specified via myst.yml are [missing alt text](https://dequeuniversity.com/rules/axe/4.11/image-alt?application=AxeChrome). Is it possible to add a myst.yml config property for the logo alt text that would apply the alt attribute as needed? Thank you!

Steps to reproduce:
1. Download the axe DevTools Chrome extension
2. Visit a page generated with book theme in Chrome
3. Open the Chrome Developer Tools panel
4. Click through to the axe DevTools tab
5. Run the free Full Page Scan
6. See the "Images must have alternative text" warning
